extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -579,8 +599,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
             : this.refreshManager_ || new RefreshManager(this, {
```

Comment:
Any reason this wasn't always the following (I think it will make it more readable).
this.refreshManager_ = this.refreshManager_ || ...

Reply:
Done.